core/src/test/java/org/testcontainers/containers/ReusabilityUnitTests.java

```diff
@@ -149,6 +153,26 @@ public void shouldReuseIfListReturnsID() {
             Mockito.verify(client, Mockito.never()).startContainerCmd(existingContainerId);
         }
 
+        @Test
+        public void shouldSetLabelsIfEnvironmentDoesNotSupportReuse() {
+            // TODO mock TestcontainersConfiguration
```

Comment:
TODO now or later?

Reply:
Copied from others - need to think (separately) how to properly do it